OpenSim/Common/Component.h

```diff
@@ -1256,9 +1256,15 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Component, Object);
 
     /**
      * %Set all values of the state variables allocated by this Component.
-     * Includes state variables allocated by its subcomponents.
+     * Includes state variables allocated by its subcomponents. Note, this
+     * method simply sets the values on the input State. If other conditions
+     * must be met (such as fiber and tendon equilibrium for muscles, or to
+     * satisfy kinematic constraints for Coordinates) you must invoke the
+     * appropriate methods on Model (e.g. assemble() to satisfy constraints or
+     * equilibrateMuscles()) to satisfy these conditions starting from the
```

Comment:
Use the same ordering within the parentheses outside the parentheses ((1) muscles (2) coordinates).